baseline/OSPS-VM.yaml

```diff
@@ -284,16 +151,109 @@ controls:
+      dependencies.
+      Ensure that violations of your SCA policy are addressed before software
+      is merged as well as before it releases, reducing the risk of compromised
+      delivery mechanisms or released software assets that are vulnerable or
+      malicious.
+    family: Vulnerability Management
+    mappings:
+      - reference-id: BPB
+        identifiers:
+          - Q-B-12
+          - Q-S-9
+          - S-B-14
+          - S-B-15
+          - A-B-3
+          - A-B-8
+      - reference-id: CRA
+        identifiers:
+          - 1.2a
+          - 1.2b
+          - 1.2c
+          - 2.1
+          - 2.2
+          - 2.3
+      - reference-id: SSDF
+        identifiers:
+          - PO.4
+          - PW1.2
+          - PW8.1
+          - RV2.1
+          - RV 2.2
+      - reference-id: CSF
+        identifiers:
+          - GV.RM-05
+          - GV.RM-06
+          - GV.PO-01
+          - GV.PO-02
+          - ID.RA-01
+          - ID.RA-08
+          - ID.IM-02
+      - reference-id: OC
+        identifiers:
+          - 4.1.5
+          - 4.2.1
+          - 4.3.2
+      - reference-id: OCRE
+        identifiers:
+          - 124-564
+          - 832-555
+          - 611-158
+          - 207-435
+          - 088-377
+    assessment-requirements:
+      - id: OSPS-VM-05.01
+        text: |
+          The project documentation MUST include a policy that defines a threshold
+          for remediation of SCA findings related to vulnerabilities and licenses.
+        applicability:
+          - Maturity Level 3
+        recommendation: |
+          Document a policy in the project that defines a threshold for
+          remediation of SCA findings related to vulnerabilities and licenses.
+          Include the process for identifying, prioritizing, and remediating
+          these findings.
+      - id: OSPS-VM-05.02
+        text: |
+          The project documentation MUST include a policy to address SCA
+          violations prior to any release.
+        applicability:
+          - Maturity Level 3
+        recommendation: |
+          Document a policy in the project to address applicable Software
+          Composition Analysis results before any release, and add status checks
+          that verify compliance with that policy prior to release.
+      - id: OSPS-VM-05.03
+        text: |
+          All proposed changes to the project's codebase must be automatically
+          evaluated against a documented policy for known vulnerabilities and
+          blocked in the event of violations except when declared and suppressed
+          as non-exploitable.
+        applicability:
+          - Maturity Level 3
+        recommendation: |
+          Create a status check in the project's version control system that
+          runs a Static Application Security Testing (SAST) tool on all changes
+          to the codebase. Require that the status check passes before changes
+          can be merged.
```

Comment:
The related policy above for OSPS-VM-05.03 requires a documented policy for known vulnerabilities but this recommendation described SAST testing to identify known vulnerabilities. I would have presumed SCA testing for dependency analysis, but I usually do not have industry colleagues describe SAST and SCA equivalent. Is this the intent?

Reply:
I think you caught me in an oversight 🤗